modules/delegates/components/DelegateVoteHistory.tsx

```diff
@@ -0,0 +1,20 @@
+import { PollVoteHistoryList } from 'modules/polls/components/PollVoteHistoryList';
+import { AddressAPIStats } from 'modules/address/types/addressApiResponse';
+import { Box, Divider, Text } from 'theme-ui';
+import { Delegate } from '../types';
+
+export function DelegateVoteHistory({ delegate, stats }: { delegate: Delegate, stats: AddressAPIStats}): React.ReactElement {
```

Comment:
Is `delegate` prop needed here?

Reply:
It will, because we need to add more components like the MKR delegated chart over time and so on. Probably those components can load asynchronously by calling the API with the delegate address